<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhiteBoard</class>
 <widget class="QMainWindow" name="WhiteBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WhiteBoard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QRadioButton" name="radioButton_choose">
        <property name="text">
         <string>手</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_rect">
        <property name="text">
         <string>矩形</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_circle">
        <property name="text">
         <string>圆</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsView_whiteBoard"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_attribute" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_style">
            <attribute name="title">
             <string>样式</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>线宽：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_lineWidth">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="suffix">
                  <string> px</string>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>线形：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_lineType">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>实线</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>虚线</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>线色：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_boundingColor">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>填充：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_fillColor">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_text">
            <attribute name="title">
             <string>文本</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>字体：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_fontType">
                 <item>
                  <property name="text">
                   <string>微软雅黑</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>宋体</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>字号：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_fontSizze">
                 <item>
                  <property name="text">
                   <string>1px</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2px</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>重做</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>撤销</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_export"/>
    <addaction name="action_import"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_export">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/source/export.png</normaloff>:/source/export.png</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_import">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/source/import.jpeg</normaloff>:/source/import.jpeg</iconset>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="action_rect">
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="action_circle">
   <property name="text">
    <string>圆形</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>comboBox_lineType</tabstop>
  <tabstop>pushButton_boundingColor</tabstop>
  <tabstop>pushButton_fillColor</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>comboBox_fontSizze</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
